fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/DataRegistry.java

```diff
@@ -90,4 +89,64 @@ public int hashCode() {
       return java.util.Objects.hash(object, action);
     }
   }
+
+  public static final class Entry<T extends Data> {
```

Comment:
Should this be private ?

Reply:
I changed the visibility to private because it was only used inside the DataRegistry